docs/spec/api.md

```diff
@@ -1,8 +1,13 @@
-<!--GITHUB
```

Comment:
I've added draft=true to the page. It won't build into the site. If you change your mind remove that and it will build.